[2023-12-18T19:36:27.619+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_etl.traffic_etl manual__2023-12-18T19:36:26.544214+00:00 [queued]>
[2023-12-18T19:36:27.627+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_etl.traffic_etl manual__2023-12-18T19:36:26.544214+00:00 [queued]>
[2023-12-18T19:36:27.628+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-18T19:36:27.639+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): traffic_etl> on 2023-12-18 19:36:26.544214+00:00
[2023-12-18T19:36:27.647+0000] {standard_task_runner.py:57} INFO - Started process 1326 to run task
[2023-12-18T19:36:27.650+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_etl', 'traffic_etl', 'manual__2023-12-18T19:36:26.544214+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/traffic_ETL.py', '--cfg-path', '/tmp/tmpe7zw4g0l']
[2023-12-18T19:36:27.653+0000] {standard_task_runner.py:85} INFO - Job 259: Subtask traffic_etl
[2023-12-18T19:36:27.693+0000] {task_command.py:416} INFO - Running <TaskInstance: traffic_etl.traffic_etl manual__2023-12-18T19:36:26.544214+00:00 [running]> on host 8833c94e6466
[2023-12-18T19:36:27.914+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='traffic_etl' AIRFLOW_CTX_TASK_ID='traffic_etl' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T19:36:26.544214+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-18T19:36:26.544214+00:00'
[2023-12-18T19:36:29.731+0000] {logging_mixin.py:154} INFO - Data loaded from /opt/***/dags/data/traffic111.csv
[2023-12-18T19:36:29.914+0000] {warnings.py:109} WARNING - /opt/***/dags/traffic_ETL.py:76: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc['日期'] = df_loc['日期'].astype('str')

[2023-12-18T19:36:29.927+0000] {warnings.py:109} WARNING - /opt/***/dags/traffic_ETL.py:77: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc['日期'] = pd.to_datetime(df_loc['日期'], format='%Y%m%d')

[2023-12-18T19:36:29.967+0000] {warnings.py:109} WARNING - /opt/***/dags/traffic_ETL.py:78: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc[['年', '月', '日']] = df_loc['日期'].dt.strftime('%Y-%m-%d').str.split('-', expand=True)

[2023-12-18T19:36:29.973+0000] {warnings.py:109} WARNING - /opt/***/dags/traffic_ETL.py:78: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc[['年', '月', '日']] = df_loc['日期'].dt.strftime('%Y-%m-%d').str.split('-', expand=True)

[2023-12-18T19:36:29.992+0000] {warnings.py:109} WARNING - /opt/***/dags/traffic_ETL.py:78: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_loc[['年', '月', '日']] = df_loc['日期'].dt.strftime('%Y-%m-%d').str.split('-', expand=True)

[2023-12-18T19:36:30.076+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-18T19:36:30.108+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_etl, task_id=traffic_etl, execution_date=20231218T193626, start_date=20231218T193627, end_date=20231218T193630
[2023-12-18T19:36:30.193+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-18T19:36:30.226+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
